dependency_manager/src/edm_tool/bazel.py

```diff
@@ -0,0 +1,55 @@
+"Bazel related functions for edm_tool."
+import sys
+
+import yaml
+
+
+def generate_deps(args):
+    "Parse the dependencies.yaml and print content of *.bzl file to stdout."
+    with open(args.dependencies_yaml, 'r', encoding='utf-8') as f:
+        deps = yaml.safe_load(f)
+    print("Buid files: ", args.build_file, file=sys.stderr)
+    if args.build_file:
+        build_files = dict((f.split(":")[1], f) for f in args.build_file)
+    else:
+        build_files = {}
+
+    print('load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")')
+    print(
+        'load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")'
+    )
+    print()
+    print()
+
+    print("def edm_deps():")
```

Comment:
Consistency wise, I would prefer the usage of `print(f""" ..` like you did below.

Reply:
OK